### ISSUE TITLE
exports regression between NodeJS 12 and 13

### ISSUE THREAD

**Issue by MrArnoldPalmer** [eemeli/yaml#140]
Minimal Reproduction: https://github.com/MrArnoldPalmer/yaml-modules-test

Hey there,

Users of aws-cdk recently reported an issue when using cdk with nodejs >= 13.0.0 < 13.7.0. I spent some time narrowing it down to `--experimental-modules` support in yaml 1.8.0.

The repository above illustrates the issue. When run with NodeJS 12, there is no issue. With node 13.0.0-13.6.0, it fails without `--experimental-modules` turned on. I'm not quite sure if there is a way to have this nested export to continue to with work CJS `require` statements in newer versions of node.

```bash
 - valinm - ~/dev/yaml-test > nvm use 12
Now using node v12.16.1 (npm v6.13.4)
 - valinm - ~/dev/yaml-test > node index.js
SUCCESS
 - valinm - ~/dev/yaml-test > nvm use 13.6.0
Now using node v13.6.0 (npm v6.13.4)
 - valinm - ~/dev/yaml-test > node index.js
internal/modules/cjs/loader.js:628
  throw e;
  ^

Error: Package exports for '/Users/valinm/dev/yaml-test/node_modules/yaml' do not define a valid './types' target
    at resolveExportsTarget (internal/modules/cjs/loader.js:621:9)
    at applyExports (internal/modules/cjs/loader.js:502:14)
    at resolveExports (internal/modules/cjs/loader.js:551:12)
    at Function.Module._findPath (internal/modules/cjs/loader.js:657:22)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:960:27)
    at Function.Module._load (internal/modules/cjs/loader.js:855:27)
    at Module.require (internal/modules/cjs/loader.js:1033:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/Users/valinm/dev/yaml-test/index.js:1:19)
    at Module._compile (internal/modules/cjs/loader.js:1144:30) {
  code: 'MODULE_NOT_FOUND'
}
 - valinm - ~/dev/yaml-test > node --experimental-modules index.js
SUCCESS
(node:44468) ExperimentalWarning: Conditional exports is an experimental feature. This feature could change at any time
 - valinm - ~/dev/yaml-test > nvm use 13.7.0
Now using node v13.7.0 (npm v6.13.6)
 - valinm - ~/dev/yaml-test > node index.js
SUCCESS
(node:45841) ExperimentalWarning: Conditional exports is an experimental feature. This feature could change at any time
```

It doesn't look like the `yaml/types` exports are exposed through the top level `yaml` export, is there a way to change the export from our side so our users don't have to use `--experimental-modules`?

**Comment by eemeli** [comment]
Bother. I didn't test within that version range. Released 1.8.1, which renames the "require" export targets as "default", which appears to be supported unflagged by at least 13.6. I'm in transit atm, so may not be able to verify on earlier versions until later.

**Comment by MrArnoldPalmer** [comment]
Ty, I'll verify this when I can as well. Appreciate the quick response.

**Comment by eemeli** [comment]
Mobile internet was faster than I expected. 1.8.1 appears to work in 13.2.0 and later, but still fails for 13.0.0, 13.0.1 & 13.1.0.

**Comment by eemeli** [comment]
Found a fallback solution that works in 13.0 and 13.1, and released 1.8.2. Shouldn't have been so quick to push out the first patch release. :/
